tests/test_create_transactions.py

```diff
@@ -0,0 +1,131 @@
+
+from datetime import date, datetime
+from decimal import Decimal
+import unittest
+
+from trytond.pool import Pool
+import trytond.tests.test_tryton
+from trytond.tests.test_tryton import ModuleTestCase
+from trytond.tests.test_tryton import with_transaction
+from trytond.transaction import Transaction
+
+from . import factories
+
+
+class CreateTransactionsTestCase(ModuleTestCase):
+    'Test Create Transactions module'
+    module = 'create_transactions'
+
+    @with_transaction(0)
```

Comment:
Is user=0 really necessary? Better use the default user=1 and maybe wrap the few instructions that require the root user in a context manager
```
with Transaction().set_user(0):
  # ...
```
but most of the times this can be naturally solved by simply putting the company id in the transaction context!